docs/source/templates/video_classification.md

```diff
@@ -14,32 +14,77 @@ If you want to build a video classification machine learning model, for example
 
 You can build a video classifier using the HyperText tag or the Video tag.
 
-## Template Preview
-
-Interactively preview this labeling template:
+## Interactive Template Preview
 
 <div id="main-preview"></div>
```

Comment:
I think we should remove this previews for video, I don't know when we will add support for them.